### PR TITLE
DDBTEAM-82: Fixed ad login

### PR DESCRIPTION
**Pull request by cableman**
See https://github.com/eReolen/base/pull/59

### CODE REVIEW

File: breol_base/js/breol_base_adgangsplatformen.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  Drupal.behaviors.breol_base_adgangsplatformen = {
+    attach: function (context, settings) {
+      $('.adgangsplatformen-button').once(function() {
+        var element = $(this);
+        var path = window.location.pathname;
+        if (path === "/") {
+          path = "/user";
+        }
+        element.attr('href', element.attr('href') + '?destination=' + path);
```
Comment (rimi-itk) [inline breol_base/js/breol_base_adgangsplatformen.js]: Inconsistent use of quotes in file (both `'` and `"` are used).

File: ding_unilogin/ding_unilogin.module [hunk 2]
```diff
@@ -240,6 +248,10 @@ function ding_unilogin_publizon_client_id_alter(&$client_id) {
 
 /**
  * Whether to globally log out.
+ *
+ * @param null $state
```
Comment (rimi-itk) [inline ding_unilogin/ding_unilogin.module]: I suppose that `$state` can be something other that `null`.

File: breol_base/js/breol_base_adgangsplatformen.js (hunk 1 above)
Comment (cableman) [reply breol_base/js/breol_base_adgangsplatformen.js]: Fixed.

File: ding_unilogin/ding_unilogin.module (hunk 2 above)
Comment (cableman) [reply ding_unilogin/ding_unilogin.module]: Fixed.

**Pull request merged**